website/config/algolia/crawler-settings.js

```diff
@@ -0,0 +1,129 @@
+new Crawler({
```

Comment:
Note that this file was dumped from the search console after a bit of tweaking to improve rankings.  To see the actual changes made, see https://github.com/cursorless-dev/cursorless/pull/915/commits/b37cd1619509d175087058a7f63df4483d2163bc